pytorch_lightning/utilities/device_parser.py

```diff
@@ -137,6 +138,27 @@ def parse_tpu_cores(tpu_cores: Optional[Union[int, str, List[int]]]) -> Optional
     return tpu_cores
 
 
+def parse_cpu_cores(cpu_cores: Optional[Union[int, str, List[int]]]) -> Optional[int]:
```

Comment:
why is the input typed as optional if CPUAccelerator.parse_devices doesn't accept optional inputs?

Reply:
good catch!